packages/ui/src/theme/createTheme/createComponentCSS.ts

```diff
@@ -1,4 +1,4 @@
-import { DefaultTheme, WebTheme } from '../types';
+import { WebTheme } from '../types';
```

Comment:
Linting fix due to new rules ([ref](https://github.com/aws-amplify/amplify-ui/pull/5830/files#diff-6e97cd3294540e00be8979d5d20d4f092e2d49d3d7e796d472f3e03961d69ab5R31))